sound/soc/sof/intel/hda-ipc.h

```diff
@@ -53,4 +53,10 @@ int cnl_ipc_send_msg(struct snd_sof_dev *sdev, struct snd_sof_ipc_msg *msg);
 void cnl_ipc_dump(struct snd_sof_dev *sdev);
 void cnl_ipc4_dump(struct snd_sof_dev *sdev);
 
+irqreturn_t mtl_ipc_irq_thread(int irq, void *context);
+int mtl_ipc_send_msg(struct snd_sof_dev *sdev, struct snd_sof_ipc_msg *msg);
+void mtl_ipc_dump(struct snd_sof_dev *sdev);
+int mtl_dsp_ipc_get_mailbox_offset(struct snd_sof_dev *sdev);
+int mtl_dsp_ipc_get_window_offset(struct snd_sof_dev *sdev, u32 id);
```

Comment:
I understand these are all related to ipc but even then isnt it better for all these to just be in mtl.h

Reply:
I just added those here because the lines above add the cnl_ stuff. But indeed why this is in the generic file escapes me. We should probably move the cnl stuff in a new cnl.h. But that's problematic for backports as well, so maybe move these mtl things and forget about cnl?